### PR TITLE
[JENKINS-59613] Rename all occurrences of "CloudBees Jenkins Advisor" by "Jenkins Health Advisor by CloudBees"

### PR DESCRIPTION
**Pull request by aheritier**
[https://issues.jenkins-ci.org/browse/JENKINS-59613](https://issues.jenkins-ci.org/browse/JENKINS-59613)

### CODE REVIEW

**Comment by aheritier** [comment]
I can add you as review @Evildethow 

https://github.com/jenkins-infra/repository-permissions-updater/pull/1291

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ config.save()
 
 ### System Properties
 
-#### CloudBees Jenkins Advisor Upload Recurrence Period
+#### Jenkins Health Advisor by CloudBees Upload Recurrence Period
```
Comment (recampbell) [inline README.md]: Just "Upload Recurrence Period"

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Overriding with Java System Property:
 -Dcom.cloudbees.jenkins.plugins.advisor.BundleUpload.recurrencePeriodHours=1
 ```
 
-#### CloudBees Jenkins Advisor Upload Timeout
+#### Jenkins Health Advisor by CloudBees Upload Timeout
```
Comment (recampbell) [inline README.md]: Just "Upload Timeout"

Comment (recampbell) [reply README.md]: Please apply same advice below. We don't need to repeat the plugin name for every setting.

Comment (aheritier) [reply README.md]: yes it makes sense

Comment (aheritier) [reply README.md]: fixed

**Pull request merged**